powersimdata/design/investment/README.md

```diff
@@ -1,72 +1,64 @@
-Note: Not all tests are done. 
-
 This adds cost calculations for all new (added since base_grid) transmission, generation, and storage capacity.
 
 `create_mapping_files.py` contains functions that help create shapefiles and map lat/lon points to regions from shapefiles. 
 `investment_costs.py` uses these mappings to find the regional cost multipliers (for both generation/storage and ac transmission).
 
 ### Fundamental calculations
-There are 3 separate cost calculations: one for dclines, one for branches, one for plants.
-
+There are 3 separate cost calculations: one for dclines, one for branches (AC transmission), one for plants.
 
-**branches: calculate_ac_inv_costs**
+#### branches: calculate_ac_inv_costs
 
 For all capacity added on a **line**, the investment cost is:
-`Cost ($2010) = rateA (MW) * lengthMi (miles) * costMWmi ($2010/MW-mi) * mult (regional cost multiplier, unitless)`
+`Cost ($today) = rateA (MW) * lengthMi (miles) * costMWmi ($2010/MW-mi) * mult (regional cost multiplier) * inflation(2010 to today)`
```

Comment:
should we write inflation($2010 to today)?

Reply:
Done.